spec/services/food_service_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'Movie Service' do
+  it 'creates a connection object'do
+    results = FoodService.conn
+    expect(results.nil?).to eq(false)
+  end
+  it 'returns search results'do
+  results = FoodService.search_foods("Sweet Potatoes")
+
+    expect(results[:totalHits]).to eq(44128)
```

Comment:
In our service we should only test the structure of how the JSON is returned, not the individual values, which are likely to change when the API changes.